code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2084,4 +2084,171 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	taste_description = "a sweet and sour slice of apple"
 	glass_icon_state = "jack_rose"
 	glass_name = "Jack Rose"
-	glass_desc = "Enough of these, and you really will start to suppose your toeses are roses."
\ No newline at end of file
+	glass_desc = "Enough of these, and you really will start to suppose your toeses are roses."
+
+/datum/reagent/consumable/ethanol/turbo
+	name = "Turbo"
+	id = "turbo"
+	description = "A turbulent cocktail associated with outlaw hoverbike racing. Not for the faint of heart."
+	color = "#e94c3a"
+	boozepwr = 85
+	quality = DRINK_VERYGOOD
+	taste_description = "the outlaw spirit"
+	glass_icon_state = "turbo"
+	glass_name = "Turbo"
+	glass_desc = "A turbulent cocktail for outlaw hoverbikers."
+
+/datum/reagent/consumable/ethanol/turbo/on_mob_add(mob/living/M)
+	..()
+	M.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown=-0.20, blacklisted_movetypes=(FLYING|FLOATING))
+
+/datum/reagent/consumable/ethanol/turbo/on_mob_delete(mob/living/M)
+	M.remove_movespeed_modifier(id)
+	..()
+
+
+/datum/reagent/consumable/ethanol/turbo/on_mob_life(mob/living/carbon/M)
+	if(prob(7))
+		to_chat(M, "<span class='notice'>[pick("You feel disregard for the rule of law.", "You want to go fast.", "You feel pumped!", "Your head is pounding.", "Your thoughts are racing..")]</span>")
+	M.adjustStaminaLoss(-8)
+	return ..()
+
+/datum/reagent/consumable/ethanol/old_timer
+	name = "Old Timer"
+	id = "old_timer"
+	description = "an archaic potation enjoyed by old coots of all ages."
+	color = "#996835"
+	boozepwr = 35
+	quality = DRINK_NICE
+	taste_description = "simpler times"
+	glass_icon_state = "old_timer"
+	glass_name = "Old Timer"
+	glass_desc = "WARNING! May cause premature aging!"
+
+/datum/reagent/consumable/ethanol/old_timer/on_mob_life(mob/living/carbon/M)
+	if(prob(20))
+		if(ishuman(M))
+			var/mob/living/carbon/human/N = M
+			N.age += 1
+			if(N.age > 70)
+				N.facial_hair_color = "ccc"
+				N.hair_color = "ccc"
+				N.update_hair()
+				if(N.age > 100)
+					N.become_nearsighted(id)
+					if(N.gender == MALE)
+						N.facial_hair_style = "Beard (Very Long)"
+						N.update_hair()
+
+				if(N.age > 969) //Best not let people get older than this or i might incur G-ds wrath
+					M.visible_message("<span class='notice'>[M] becomes older than any man should be.. and crumbles into dust!</span>")
+					M.dust(0,1,0)
+
+	return ..()
+
+/datum/reagent/consumable/ethanol/catsip
+	name = "Catsip"
+	id = "catsip"
+	description = "a kawaii drink from space-Japan."
+	color ="#ff99ac"
+	boozepwr = 50
+	quality = DRINK_NICE
+	taste_description = "sweet milkies"
+	glass_icon_state = "catsip"
+	glass_name = "Catsip"
+	glass_desc = "Unfortunately has a tendency to induce the perculiar vocal tics of a wapanese mutant in the imbiber."
+
+/datum/reagent/consumable/ethanol/catsip/on_mob_life(mob/living/M)
+	if(prob(8))
+		M.say(pick("Nya.", "N-nya!", "NYA!"), forced = "catsip")
```

Comment:
why

Reply:
lol

Reply:
I need a good reason to keep this.

Reply:
Well, the biggest reason should be the overwhelmingly positive feedback i have received for this drink. Since only you and oranges of like 25 people i have shown this drink to have objected, can you explain to me what the issue is?

The recipe requires corn syrup(corn starch & acid) and therefore takes some effort to make, which will prevent it being made too often.

I can lower the prob of the forced speech or make the ingredients even harder. But without you explaining what you take issue with it is hard for me to address it.

Reply:
cringe

Reply:
sorry this drink needs to go if it is going to have any wagging/nya'ing in it.

Reply:
> 
> 
> sorry this drink needs to go if it is going to have any wagging/nya'ing in it.

I am totally willing to remove the drink, as I have been all along. 

All I asked for was a justification. I expected to have to remove it in fact, i did not expect for oranges to close my PR before i could even see this post